src/unstructured_client/_hooks/custom/log_retries_after_error_hook.py

```diff
@@ -1,47 +1,55 @@
 import logging
```

Comment:
It's probably worth renaming this to logger_hook or something like that.

Reply:
Not sure what you mean by this, I saw that default `logging` was used to log some of the messages. Changed it to use custom unstructured client logger.

Reply:
Oh, I see now. Thought it was a comment for split_pdf_hook.py. Yep, I agree, should be renamed.

Reply:
Changed